docs/platforms/go/common/tracing/instrumentation/caches-module.mdx

```diff
@@ -91,4 +91,4 @@ if parentSpan != nil {
 }
 ```
 
-You should now have the right spans in place. Head over to the [Cache dashboard](https://sentry.io/orgredirect/organizations/:orgslug/performance/caches/) to see how your cache is performing.
+You should now have the right spans in place. Head over to the [Cache dashboard](https://sentry.io/orgredirect/organizations/:orgslug/insights/backend/caches) to see how your cache is performing.
```

Comment:
```suggestion
You should now have the right spans in place. Head over to the [Cache dashboard](https://sentry.io/orgredirect/organizations/:orgslug/insights/backend/caches/) to see how your cache is performing.
```

Sometimes there's weird behaviour when we don't prepend with a slash, so I suggest we add one here